packages/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -284,6 +286,10 @@ export interface ChartProps extends VictoryChartProps {
    * cases, the legend may not be visible until enough padding is applied.
    */
   legendPosition?: 'bottom' | 'bottom-left' | 'right';
+  /**
+   * @beta Text direction of the legend labels.
+   */
+  legendDirection?: 'ltr' | 'rtl';
```

Comment:
rather than a prop, should this auto switch when browser is RTL?

Reply:
Per conversation, leaving this PR as is and opened #9656 to investigate adding logic to smartly set the default.